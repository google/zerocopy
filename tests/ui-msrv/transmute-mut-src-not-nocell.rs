../ui-nightly/transmute-mut-src-not-nocell.rs