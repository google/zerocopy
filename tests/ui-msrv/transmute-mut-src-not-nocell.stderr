error[E0277]: the trait bound `Src: NoCell` is not satisfied
  --> tests/ui-msrv/transmute-mut-src-not-nocell.rs:24:35
   |
24 | const SRC_NOT_NO_CELL: &mut Dst = transmute_mut!(&mut Src);
   |                                   ^^^^^^^^^^^^^^^^^^^^^^^^ the trait `NoCell` is not implemented for `Src`
   |
note: required by `AssertSrcIsNoCell`
  --> tests/ui-msrv/transmute-mut-src-not-nocell.rs:24:35
   |
24 | const SRC_NOT_NO_CELL: &mut Dst = transmute_mut!(&mut Src);
   |                                   ^^^^^^^^^^^^^^^^^^^^^^^^
   = note: this error originates in the macro `transmute_mut` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `Src: NoCell` is not satisfied
  --> tests/ui-msrv/transmute-mut-src-not-nocell.rs:24:35
   |
24 | const SRC_NOT_NO_CELL: &mut Dst = transmute_mut!(&mut Src);
   |                                   ^^^^^^^^^^^^^^^^^^^^^^^^ the trait `NoCell` is not implemented for `Src`
   |
note: required by a bound in `AssertSrcIsNoCell`
  --> tests/ui-msrv/transmute-mut-src-not-nocell.rs:24:35
   |
24 | const SRC_NOT_NO_CELL: &mut Dst = transmute_mut!(&mut Src);
   |                                   ^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `AssertSrcIsNoCell`
   = note: this error originates in the macro `transmute_mut` (in Nightly builds, run with -Z macro-backtrace for more info)
